challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |item| item[0].capitalize + item[1..-1] }
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select { |item| item == 'dog' }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.reject { |item| item == 'dog' }
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.reduce(0) { |memo| memo += 1 }
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input.take 2
```

Comment:
@nicolasmccurdy looks good. you can also use `.first` here too, you'll see it used often in cmm's rails apps at times